src/Pages/Review/Review.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Basket from './components/Basket/Basket.jsx';
+import TicketList from './components/TicketList/TicketList.jsx';
+import Total from './components/Total/Total.jsx';
+import './Review.scss';
+
+const Review = ({ Campaign = {} }) => {
+    const { Products, title } = Campaign;
+    return (
+        <div className="flex-layout">
+            {[
+                (
+                    <TicketList
+                        key="listings"
+                        title={title}
```

Comment:
Title may be `undefined` here but its marked as a required title by `TicketList` component. Perhaps guard against undefined value in the destructing of 'Campaign' object on line 8. i.e `const { Products, title = '' } = Campaign;`